content/docs/authentication/access_tokens_guard.md

```diff
@@ -539,6 +539,20 @@ import router from '@adonisjs/core/services/router'
 const SessionController = () => import('#controllers/session_controller')
 
 router.post('session', [SessionController, 'store'])
-router.destroy('session', [SessionController, 'destroy'])
+router.delete('session', [SessionController, 'delete'])
```

Comment:
The controller method name should be `destroy` to match the resource naming convention. https://github.com/adonisjs/core/blob/6.x/stubs/make/controller/resource.stub#L44